lib/_http_client.js

```diff
@@ -47,6 +47,10 @@ const {
 } = require('internal/errors').codes;
 const { validateTimerDuration } = require('internal/timers');
 const is_reused_symbol = require('internal/freelist').symbols.is_reused_symbol;
+const {
+  DTRACE_HTTP_CLIENT_REQUEST = () => {},
```

Comment:
If we are now importing them from `internal/dtrace` and even test our internals, we can just remove the default values in the destructuring?

Reply:
done